okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -389,6 +389,10 @@ private void ead1TenantInterface(Tenant tenant, TenantInstallOptions options,
                                    ModuleDescriptor mdFrom, ModuleDescriptor mdTo,
                                    ProxyContext pc, Handler<ExtendedAsyncResult<Void>> fut) {
 
+    if (!options.getInvoke()) {
+      fut.handle(new Success<>());
+      return;
```

Comment:
Make a note that `invoke = false` will skip `purge = true`

Reply:
Done and I've added a chapter about invoke in the guide